.github/workflows/pr-plugin-build.yml

```
@@ -0,0 +1,199 @@
+        run: |
+          bash .github/scripts/generate-pr-plugin.sh \
+            "${{ steps.version.outputs.version }}" \
+            "${{ github.event.pull_request.number }}" \
+            "$(git rev-parse --short HEAD)" \
+            "webgui-pr-${{ steps.version.outputs.version }}.tar.gz" \
+            "${{ steps.upload-txz.outputs.txz_url }}"
+      
+      - name: Upload PLG to R2
+        if: steps.changed-files.outputs.has_changes == 'true'
+        id: upload-plg
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:10:28: Double quote to prevent globbing and word splitting

(shellcheck)